xyz.ayrsd.AudioStellar.yml

```diff
@@ -0,0 +1,319 @@
+app-id: xyz.ayrsd.AudioStellar
```

Comment:
`ayrsd.xyz` doesn't exist. therefor the app id doesn't meet the requirements.

Reply:
oh... i have the audiostellar.xyz domain . should i rename to xyz.audiostellar.AudioStellar ?

Reply:
that's what the documentation say. And you even acknowledged it.

![image](https://github.com/flathub/flathub/assets/114441/4c762e11-8937-48e2-afbf-854e2c3c9c4b)